.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: deploy
+
+on: [pull_request]
+
+env:
+    DEPLOY_USER: omniledger
+    DEPLOY_HOST: srv1.c4dt.org
+
+jobs:
+    deploy-website:
```

Comment:
Remind me how it is assured that this is only run if the tests pass?

Reply:
The `if: ${{ github.event.workflow_run.conclusion == 'success' }}` and the deps on another workflow ensure that.